modules/scale-down-data-plane.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-managing.adoc
+
+:_content-type: PROCEDURE
+[id="scale-down-data-plane_{context}"]
+= Scaling down the data plane to zero
+
+If you are not using the hosted control plane, to save the resources and cost you can scale down a data plane to zero.
+
+.Procedure
+
+. Set the `kubeconfig` file to access the hosted cluster by running the following command:
++
+[source,terminal]
+----
+$ export KUBECONFIG=<install_directory>/auth/kubeconfig
+----
+
+. Get the name of `NodePool` associated to your hosted cluster by running the following command:
++
+[source,terminal]
+----
+$ oc get nodepool --namespace <HOSTED_CLUSTER_NAMESPACE>
+----
+
+. Optional: To prevent the pods from draining, add the `nodeDrainTimeout` field in the `NodePool` resource by running the following command:
++
+[source,terminal]
+----
+$ oc edit NodePool <nodepool> -o yaml --namespace <HOSTED_CLUSTER_NAMESPACE>
```

Comment:
fwiw you can also `oc scale NodePool <nodepool> --namespace <HOSTED_CLUSTER_NAMESPACE> --replicas 0`

Reply:
ah nvm I see you have that below.